app/templates/events/view/tickets/index.hbs

```diff
@@ -22,7 +22,7 @@
       </tr>
       <tr>
         {{#link-to 'events.view.tickets.orders.list' 'cancelled' tagName='td'
-        class='ui grey inverted segment'}}
+        class='ui grey inverted segment links'}}
```

Comment:
Can we keep the classname something other than 'links'? I mean, somehow it may be misunderstood that there are links as its children who will be given classes 'link'. @geekyd